File: LeetCode/p01_Two_Sums/TwoSums.ts
function twoSums(nums: number[], target: number): number[]{
    for(let i:number = 0; i < nums.length; i++){
        for(let j: number = i + 1; j < nums.length; j++){
            if(nums[j] == target - nums[i]!){
                return [i, j];
            }
        }
    }
    return[];
}

File: LeetCode/p01_Two_Sums/twoSums.kt
package problemSolving.LeetCode

class Solution{
    fun TwoSums(nums: Array<Int>, target: Int): IntArray{
        for(i in 0 until nums.size){
            var j = i + 1
            for(j in 0 until nums.size){
                if(nums[i] + nums[j] == target){
                    return intArrayOf(i, j)
                }
            }
        }
        return intArrayOf(0, 0)   // values not found
    }
}